src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -270,6 +274,9 @@ private List<Track> GetTracks(Artist artist, ParsedTrackInfo parsedTrackInfo)
                 return new List<Track>();
             }
 
+            parsedTrackInfo.AlbumTitle = Parser.CleanAlbumTitle(parsedTrackInfo.AlbumTitle);
+            _logger.Debug("Cleaning Album title of common matching issues. Cleaned album title is '{0}'", parsedTrackInfo.AlbumTitle);
```

Comment:
Do we need to clean here if you already clean it in the Parser.cs `ParseMusicPath` function called above?

Reply:
Or Maybe we just clean here and not needed in the ParseMusicPath function?